test/examples/basic.docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  collector:
+    # hopefully 0.27.0 will include polling exporter
```

Comment:
Current version is 0.35.0, is it available now?

Reply:
I don't think so. The PR is still open: https://github.com/signalfx/splunk-otel-collector/pull/436